### DISCUSSION TITLE
A new Home

### DISCUSSION THREAD

**Discussion by Trouffman** [DistroAV/DistroAV#859] [General]
It is a period of great activity @paulpv developing, PR-ing from a hidden base, managed to push the first PR against the main repo branch.

During this dev phase @Palakis approved the code and we can expect a new set of binaries in the coming days fixing the dreadful audio bug that plagged the 4.10 release.

The road ahead is bright with activity picking up and following suggestion this project has been moved to it's own repo at github.com/obs-ndi.

This project is maintained by volunteers meaning dev, support & help contribution are greatly encouraged.

Happy to see the project moving!

**Comment by paulpv** [comment]
https://github.com/obs-ndi/obs-ndi/releases/tag/4.11.0 is released!

**Comment by jbcurler2010** [comment]
Nice work!!!

John Benton
Curling Stadium
612-202-4588
